vcluster/configure/vcluster-yaml/sleep-mode.mdx

```diff
@@ -187,6 +188,17 @@ To configure sleep mode for the ingress controller, make sure it stays responsiv
 <SleepModeIngressExample />
 </details>
 
+### Istio gateway
+
+With Istio installed on the host cluster and the [Istio integration](./integrations/istio) enabled, [Gateway](https://istio.io/latest/docs/reference/config/networking/gateway/) and [VirtualService](https://istio.io/latest/docs/reference/config/networking/virtual-service/) resources created in the vCluster
```

Comment:
This implies that the integration is required for this to work, is that true ? If so we should clarify.
If it's not true, is there anything we have to do to turn it on with vanilla istio ? Or is the sleep controller smart enough to figure out ingress-nginx vs istio ingress.

Reply:
It is true, so I'll call that out more explicitly.  WIthout the integration enabled, the sleep mode controller isn't started.  Otherwise it would be making modifications to Istio resources in the virtual cluster that never get synced/translated to the host.